01_fib/n0npax/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"math"
+	"os"
+)
+
+// Allow to overwtite stream by tests
+var out io.Writer = os.Stdout
+
+// This function is useless and it's provided just due to requirements
+func main() {
+	fib(7)
+}
+
+// Wrapper function to print computed Fibonacci numbers
+func fib(num int) {
+	for _, v := range dispatchFib(num) {
```

Comment:
If you insist on creating an array you could use a one-liner here if you want
`fmt.Fprintln(strings.Join(dispatchFib(num), "\n"))`

Reply:
`strings.Join` definition looks like: `func Join(a []string, sep string) string`. https://golang.org/pkg/strings/#Join

In this case `dispatchFib(num)` returns `[]int`.

Reply:
Oh! of course, apologies!